NEWS.md

```diff
@@ -1,89 +1,109 @@
-# ggplot2 3.0.0.9000
+# ggplot2 3.1.0
 
-*   `geom_boxplot()` now understands the `width` parameter even when used with
-    a non-standard stat, such as `stat_identity()` (@clauswilke, #2893).
-
-*   `scale_*_date()`, `scale_*_time()` and `scale_*_datetime()` can now display 
-    a secondary axis that is a __one-to-one__ transformation of the primary axis,
-    implemented using the `sec.axis` argument to the scale constructor 
-    (@dpseidel, #2244).
+## Breaking changes
 
-*   The error message in `compute_aesthetics()` now provides the names of only
-    aesthetics with mismatched lengths, rather than all aesthetics (@karawoo,
-    #2853).
-    
-*   `coord_sf()` now respects manual setting of axis tick labels (@clauswilke,
-    #2857, #2881).
-    
-*   `coord_sf()` now accepts two new parameters, `label_graticule` and `label_axes`,
-    that can be used to specify which graticules to label on which side of the plot
-    (@clauswilke, #2846).
+This is a minor release and breaking changes have been kept to a minimum. End users of ggplot2 are unlikely to encounter any issues. However, there are a few items that developers of ggplot2 extensions should be aware of. For additional details, see also the discussion accompanying issue #2890.
+
+*   In non-user-facing internal code (specifically in the `aes()` function and in
+    the `aesthetics` argument of scale functions), ggplot2 now always uses the British
+    spelling of aesthetics containing the word "colour". When users specify a "color"
+    aesthetic it is automatically renamed to "colour". This principle is also applied
```

Comment:
Simplify the "This principle is also…" to "This also applies to non-standard aesthetics…" ?
(stylistic, it's fine either way)

Reply:
I normally try to avoid using "this" as a pronoun (as opposed to a modifier), as it leads to imprecise writing. But I agree that "this principle" is awkward, so maybe imprecise is better here.

Reply:
I think "this renaming" works better than just "this":
> When users specify a "color" aesthetic it is automatically renamed to "colour". This renaming is also applied to non-standard aesthetics that contain the word "color".
versus:
> When users specify a "color" aesthetic it is automatically renamed to "colour". This is also applied to non-standard aesthetics that contain the word "color".

Reply:
Sure. 👍